### ISSUE TITLE
Cronjob creates a job but no pod is created to run it

### ISSUE THREAD

**Issue by mleneveut** [kubernetes/kubernetes#68362]
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind bug

**What happened**:
I created a cronjob. It creates a job, but no pod is created to actually run the job.

**What you expected to happen**:
A pod is created every hour as scheduled.

**How to reproduce it (as minimally and precisely as possible)**:
I pushed the descheduler v0.7.0 image in my Azure Container Registry.

I created a configmap for the descheduler policy : 

```
apiVersion: v1
kind: ConfigMap
metadata:
  name: descheduler
  namespace: kube-system
data:
  policy.yaml: |-  
    apiVersion: descheduler/v1alpha1
    kind: DeschedulerPolicy
    strategies:
      RemoveDuplicates:
         enabled: false
      LowNodeUtilization:
         enabled: true
         params:
           nodeResourceUtilizationThresholds:
             thresholds:
               cpu: 20
               memory: 20
               pods: 20
             targetThresholds:
               cpu: 40
               memory: 40
               pods: 40
      RemovePodsViolatingInterPodAntiAffinity:
        enabled: true
      RemovePodsViolatingNodeAffinity:
        enabled: true
        params:
          nodeAffinityType:
          - requiredDuringSchedulingIgnoredDuringExecution
```

And created the cronjob : 

```
apiVersion: batch/v1beta1
kind: CronJob
metadata:
  name: descheduler
  namespace: kube-system
spec:
  schedule: "5 * * * *"
  jobTemplate:
    metadata:
      name: descheduler
      annotations:
        scheduler.alpha.kubernetes.io/critical-pod: "true"
    spec:
      template:
        spec:
          containers:
          - name: descheduler
            image: mycr.azurecr.io/descheduler:v0.7.0
            volumeMounts:
            - mountPath: /policy-dir
              name: policy-volume
            command:
            - /bin/descheduler
            - --v=4
            - --max-pods-to-evict-per-node=10
            - --policy-config-file=/policy-dir/policy.yaml
          restartPolicy: "OnFailure"
          volumes:
          - name: policy-volume
            configMap:
              name: descheduler
          imagePullSecrets:
          - name: azure-registry
```

The cronjob is created and active, a job is created : 

```
NAME          SCHEDULE    SUSPEND   ACTIVE    LAST SCHEDULE   AGE
descheduler   5 * * * *   False     1         13m             1h
NAME                     DESIRED   SUCCESSFUL   AGE
descheduler-1536242700   1         0            13m
```

Describe of the cronjob says : `Normal  SuccessfulCreate  2m    cronjob-controller  Created job descheduler-1536242700`

But no pod is created to actually run the job : 

```
k get pods --all-namespaces -a | grep descheduler
> returns no results
```

**Anything else we need to know?**:
It works when I create directly the job : 

```
apiVersion: batch/v1
kind: Job
metadata:
  name: descheduler-job
  namespace: kube-system
spec:
  parallelism: 1
  completions: 1
  template:
    metadata:
      name: descheduler
      annotations:
        scheduler.alpha.kubernetes.io/critical-pod: "true"
    spec:
      containers:
      - name: descheduler
        image: mycr.azurecr.io/descheduler:v0.7.0
        volumeMounts:
        - mountPath: /policy-dir
          name: policy-volume
        command:
        - /bin/descheduler
        - --v=4
        - --max-pods-to-evict-per-node=10
        - --policy-config-file=/policy-dir/policy.yaml
      restartPolicy: "OnFailure"
      volumes:
      - name: policy-volume
        configMap:
          name: descheduler
      imagePullSecrets:
      - name: azure-registry
```

```
k get pods --all-namespaces -a | grep descheduler
descheduler-job-2fmzr                   0/1       Completed   0          6s
```

Is there a way to see some logs of the scheduling of the pod creation ?

**Environment**:
- Kubernetes version (use `kubectl version`): 
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.2", GitCommit:"bb9ffb1654d4a729bb4cec18ff088eacc153c239", GitTreeState:"clean",
BuildDate:"2018-08-07T23:17:28Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.2", GitCommit:"bb9ffb1654d4a729bb4cec18ff088eacc153c239", GitTreeState:"clean",
BuildDate:"2018-08-07T23:08:19Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
- Cloud provider or hardware configuration: Azure AKS

**Comment by mleneveut** [comment]
@kubernetes/sig-scheduling-bugs

**Comment by mleneveut** [comment]
My bad, sorry, I first tried with cron "*/30 * * * *" and it was not working. But we have to wait for 30 minutes before the first job is ran. I thought it would launch a first one immediatly, and the other one 30 minutes after.

I then tried with "5 * * * *" and waited for my clock to reach 16h05, but perhaps I didn't wait enough.

With "*/1 * * * *" I see the pod being created every minute, so it works.
